### DISCUSSION TITLE
Stable Docker tag

### DISCUSSION THREAD

**Discussion by ekiel** [ArchiveBox/ArchiveBox#586] [General]
Is there a "known" stable Docker tag that we should be using?

**Comment by pirate** [comment]
`:latest` was briefly unstable as we were experimenting with building more frequently (on every commit to `master`), but I changed that behavior back because it was too volatile so `:latest` should be stable now (it will reflect only the latest publicly released version on https://github.com/ArchiveBox/ArchiveBox/releases).

**Reply by marleyjaffe** [reply]
The [linked](https://github.com/ArchiveBox/ArchiveBox/blob/master/docker-compose.yml) docker-compose file is using the tag `:master`, is this correct or should we be using `:latest` as our default tag? I was unable to find any documentation of the different docker tags and how they vary.

**Reply by pirate** [reply]
`:master` is correct,  `:latest` is the most recent beta/potentially unstable build.
